libs/cli/langchain_cli/package_template/README.md

```diff
@@ -1 +1,35 @@
 # __package_name_last__
+
+TODO: What does this package do
+
+## Environment Setup
+
+TODO: What environment variables need to be set (if any)
+
+## Usage
+
+To use this package, you should first have the LangChain CLI installed:
+
+```shell
+pip install -U "langchain-cli[serve]"
+```
+
+To create a new LangChain project and install this as the only package, you can do:
+
+```shell
+langchain serve new my-app
+cd my-app
+langchain serve add __package_name_last__
+```
+
+If you want to add this to an existing project, you can just run:
+
+```shell
+langchain serve add __package_name_last__
```

Comment:
We should add context on `server.py`, e.g., -

The app will create a FastAPI web server.

If you opt for it, the CLI will auto-generate code to add your template e.g.,:

```
Add the following to your app:

from llama2_functions import chain as llama2_functions_chain

add_routes(app, llama2_functions_chain, path="/llama2-functions")
```

Add these lines to `server.py`.

Reply:
I'll add this to CLI